src/_utils/focusVisible/FocusVisibleProvider.tsx

```diff
@@ -1,5 +1,13 @@
 import React, { createContext, useEffect, useState, useMemo, ReactNode } from 'react'
 
+const KEYCODE_TRIGGERING_FOCUS_VISIBLE = [
+  37, // Arrow left
+  38, // Arrow up
+  39, // Arrow right
+  30, // Arrow down
+  9, // Tab
+]
+
```

Comment:
There's a keycode file here, maybe it's good to add it there: https://github.com/blablacar/ui-library/blob/master/src/_utils/keycodes.ts

Reply:
Done

Reply:
[`KeyboardEvent .keyCode` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), can't we switch to [`KeyboardEvent .key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key)?

It is already [well supported](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#Browser_compatibility) and React's [SyntheticEvent](https://reactjs.org/docs/events.html) [set it for us](https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/packages/react-dom/src/events/getEventKey.js#L79) when the browser doesn't support it.